text/0192-multi-search-api.md

```diff
@@ -0,0 +1,79 @@
+# Multi-search API
+
+## 1. Summary
+
+The multi-search endpoint performs a batch of search queries as if using the [search API](./0118-search-api.md) for an index.
+
+This endpoint allows, for example:
+
+- Execute different search queries on the same index
+- Execute the same search query on different indexes
```

Comment:
```suggestion
- Execute search queries on different indexes
```

Should we just say that you can perform multiple searches on different indexes decorrelated of the search query in itself?

Since these are example, my comment might not be relevant. My first intuition though was that it was either one or the other and thus I had to use the same search query when querying on multiple indexes.

@maryamsulemani97 what do you think?

Reply:
I'm OK with the suggestion.